### DISCUSSION TITLE
Where do we get details steps of setting up development environment?

### DISCUSSION THREAD

**Discussion by ramakrse** [fastapi-users/fastapi-users#1220] [Q&A]
`We use [Hatch](https://hatch.pypa.io/latest/install/) to manage the development environment and production build. Ensure it's installed on your system`. 

I am going through hatch documentation, however, steps to set up the essential development environment will be helpful.

**Comment by frankie567** [comment]
Basically, you need to run `hatch shell`. It'll create a virtual Python environment, install the dependencies and enable it on your shell.

You can also directly run the command we defined like `hatch run lint` or `hatch run test`, it'll automatically run the required commands in the right environment.

**Comment by ramakrse** [comment]
Thanks for the quick response. I will try out your suggestions.
